tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -205,6 +205,111 @@ async function sendEventAndCheckResponse(mockNamespace = 'mocks') {
   );
 }
 
+async function sendCloudEventStructuredModeAndCheckResponse(mockNamespace = 'mocks') {
```

Comment:
As I can see, these both new function are pretty much the same as the old ones excepting some tiny parts, could you maybe refactor them in they way the duplicated code could be reused ?

Reply:
done